### PR TITLE
Add support for VSCode JavaScript React Language

### PR DESCRIPTION
**Pull request by zachgibson**

### CODE REVIEW

File: packages/vscode-css-in-js/src/extension.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function activate (context) {
       // return if there's no editor or it's not a javascript file
       if (
         !editor ||
-        !/javascript|typescript/.test(editor.document.languageId)
+        !/javascript|typescript|javascriptreact/.test(editor.document.languageId)
```
Comment (paulmolluzzo) [inline packages/vscode-css-in-js/src/extension.js:32]: This is not really a required change.

screen shot 2018-06-22 at 2 19 29 pm

**Comment by paulmolluzzo** [comment]
This isn't required or add much, so for that reason I'm going to close it. I don't want to go through the publishing process on a Friday after tbh. 😅 

Sorry, but thanks for the PR!

**Comment by zachgibson** [comment]
@paulmolluzzo I’ll have to investigate further then because this plugin *does not* work with language mode set to JavaScript React, but it does obviously with JavaScript. This is probably the problem with #16.

**Comment by paulmolluzzo** [comment]
Oh, I see why. We have to add that as another language in the `activationEvents` in the package.json. We need to add one for typescript too.

https://code.visualstudio.com/docs/extensionAPI/activation-events#_activationeventsonlanguage

If you want to make those changes and reopen the PR that would be great! I can try to publish the update tonight.

**Comment by zachgibson** [comment]
Done #21 👍

**Pull request closed without merging**